nodejs/docs/node-standard-and-best-practices.md

```diff
@@ -74,15 +74,15 @@ The model should be as simple as possible, just fields and Sequelize configurati
 
 ### 2.4- Helpers
 
-Helpful tools with **absolutley no business logic**. These include parsers, date formatters, etc. These are just auxiliary tools and should be able to be used in other projects.
+Helpful tools with **absolutley no business logic**. This includes parsers, date formatters, etc. Helpers are just auxiliary tools and should be abstract enough to able to be used in other projects.
 
```

Comment:
Keep in mind that helpers are just auxiliary tools, thus should be abstract enough to allow being implemented in other projects.